build/post_test.rb

```diff
@@ -1,8 +1,8 @@
 require "pty"
 
 commands = [
-  "rvm-exec 2.5.0 bundle update; rvm-exec 2.5.0 bundle exec rake circleci:post",
-  "rvm-exec 2.4.2 bundle update; rvm-exec 2.4.2 bundle exec rake test:coveralls"
+  "rvm-exec 2.5.1 bundle update; rvm-exec 2.5.1 bundle exec rake circleci:post",
+  "rvm-exec 2.4.4 bundle update; rvm-exec 2.4.4 bundle exec rake test:coveralls"
```

Comment:
I understand you were just updating the patch version here, but should coveralls still be executed using 2.4, or should it be run with 2.5?

Reply:
Its this way so it runs on the second build server, instead of blocking the first to do the docs and coveralls.

Reply:
Great, thanks @blowmage.

Reply:
I'll add a comment to that effect so we know next time.